promotion/src/test/java/org/cryptimeleon/incentive/promotion/reward/NutellaRewardTest.java

```diff
@@ -0,0 +1,17 @@
+package org.cryptimeleon.incentive.promotion.reward;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class NutellaRewardTest {
```

Comment:
Just realizing that our code/docu is a good hidden ad for Nutella...
Since this is a public repo, we should request some money from Ferrero :D

Reply:
Lol